### PR TITLE
css-canvas-medical | Erik Modica

### PR DESCRIPTION
**Pull request by ErikModica**

### CODE REVIEW

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Please let us know when it's done.

**Comment by ErikModica** [comment]
image
image

File: css-canvas-medical/styles.css [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+
+.product-row a {
+  color: rgb(62, 62, 163);
+}
+
+.quote {
+  font-family: 'Tiempos Headline', sans-serif;
+  font-style: italic;
+  font-weight: 300;
+  src: url('TiemposHeadline-LightItalic.woff2') format('woff2');
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:246]: The font family is already loaded, so you can leave out this `src` property.

File: css-canvas-medical/styles.css [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+
+.get-started-button {
+  background-color: rgb(41, 68, 100);
+  color: white;
+  padding: 10px 30px;
+  border-radius: 3px;
+  border-style: unset;
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:149]: I recommend consolidating your font settings like this:
```css
body,
button {
  font-family: 'Avenir', sans-serif;
}
```

That should help cut down on some duplication.

File: css-canvas-medical/styles.css [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+  src: url('avenir.woff2') format('woff2');
+  font-size: 16px;
+}
+
+.row2 {
+  padding: 3rem 0;
+}
```
Comment (thebearingedge) [inline css-canvas-medical/styles.css:69]: This is tough. There is a thing call a "hero" section. I'd use that for these areas with text over a background image. `row1`, `row2`, `row3` is not a good habit to get into. If you are having a hard time naming a section of the page, feel free to steal some of the content.

```css
.hero {

}

.hero-column {

}

.clinicians-prefer {

}
```

We want to get away from just numbering out classes, but naming is hard so we gotta get kind of creative.

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Great work so far!

**Comment by ErikModica** [comment]
most recent push was just cleaning up css and chaning the class names to be more organized.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**